docs/user-guide.rst

```diff
@@ -793,6 +803,13 @@ Cloud resources from an OIDC or SAML provider.
     https://cloud.google.com/iam/docs/workforce-identity-federation#workforce-pools-user-project
 
 
+Note that this library does not perform any validation on the token_url,
```

Comment:
Why duplicate this snippet?

Reply:
To have this in both workload and workforce section since they are already separated.